### PR TITLE
Fix some issues

### PR DESCRIPTION
**Pull request by rkuchan**
* Fixes https://github.com/influxdata/docs.influxdata.com/issues/286 by moving the server monitor page from the `Administration` section to the `Troubleshooting` section and deleting the original one in the `Troubleshooting` section. The version in the `Administration` section had more information.
* Fixes https://github.com/influxdata/docs.influxdata.com/issues/281 by updating examples in the docs that reference `0.9` as well as removing other needless mentions of version prior to `0.10`.
* Fixes https://github.com/influxdata/docs.influxdata.com/issues/251 by linking `installation.md` to the downloads page.
* Adds `metastore` and `line protocol` to the glossary (This takes care of some of https://github.com/influxdata/docs.influxdata.com/issues/218).
* Fixes https://github.com/influxdata/docs.influxdata.com/issues/280 by moving the clarification about authentication to a more relevant place in the docs.
* Fixes https://github.com/influxdata/docs.influxdata.com/issues/297 by adding `Z` to RFC3339 example.
* Fixes https://github.com/influxdata/docs.influxdata.com/issues/316 byt adding `<>` to spec.md.

### CODE REVIEW

**Comment by rkuchan** [comment]
@gunnaraasen, @beckettsean, does this look OK? Most of the changes are pretty small.

File: content/influxdb/v0.10/concepts/api.md [hunk 1]
```diff
@@ -15,18 +15,18 @@ All current API Endpoints are listed below.
...
 
 ```bash
 $ curl -sl -I localhost:8086/ping
 
 HTTP/1.1 204 No Content
-Request-Id: 78addfb1-5335-11e5-87f5-000000000000
-X-Influxdb-Version: 0.9.5-nightly-548b898
-Date: Wed, 21 Oct 2015 16:29:02 GMT
+Request-Id: 7d641f0b-e23b-11e5-8005-000000000000
+X-Influxdb-Version: 0.10.1
```
Comment (gunnaraasen) [inline content/influxdb/v0.10/concepts/api.md]: Update to 0.10.3

File: content/influxdb/v0.10/administration/authentication_and_authorization.md [hunk 2]
```diff
@@ -56,6 +56,8 @@ Plugins do not currently have the ability to authenticate requests and service e
 1. Create at least one [admin user](/influxdb/v0.10/administration/authentication_and_authorization/#admin-users).
 See the [authorization section](/influxdb/v0.10/administration/authentication_and_authorization/#authorization) for how to create an admin user.
 
+    > **Note:** If you enable authentication and have no users, InfluxDB will **not** enforce authentication until you create the first admin user, and InfluxDB will only accept the [query](/influxdb/v0.10/administration/authentication_and_authorization/#create-a-new-admin-user) that creates an admin user.
```
Comment (gunnaraasen) [inline content/influxdb/v0.10/administration/authentication_and_authorization.md]: > InfluxDB will **not** enforce authentication until you create the first admin user, and InfluxDB will only accept the [query](/influxdb/v0.10/administration/authentication_and_authorization/#create-a-new-admin-user) that creates an admin user.

Make it clear that only create user queries are allowed in this case, e.g.

> InfluxDB will **not** enforce authentication and will **only** accept the [query](/influxdb/v0.10/administration/authentication_and_authorization/#create-a-new-admin-user) to create a new admin user. Authentication will be enforced as usual once there is an admin user.

File: content/influxdb/v0.10/introduction/installation.md [hunk 3]
```diff
@@ -28,6 +28,9 @@ which is located at `/etc/influxdb/influxdb.conf` for default installations.
 ## Installation
 
 ### Ubuntu & Debian
+
+For `wget` instructions, please see the [downloads page](https://influxdata.com/downloads/).
```
Comment (gunnaraasen) [inline content/influxdb/v0.10/introduction/installation.md]: > For `wget` instructions,

should be

> For instructions on how to manually the Debian package,

File: content/influxdb/v0.10/introduction/installation.md [hunk 4]
```diff
@@ -54,6 +57,9 @@ sudo service influxdb start
 ```
 
 ### RedHat & CentOS
+
+For `wget` instructions, please see the [downloads page](https://influxdata.com/downloads/).
```
Comment (gunnaraasen) [inline content/influxdb/v0.10/introduction/installation.md]: > For `wget` instructions,

should be

> For instructions on how to manually the RPM package,

**Comment by gunnaraasen** [comment]
Some small changes, otherwise this is good to go!

**Pull request merged**